openid-4-verifiable-presentations-1_0.md

```diff
@@ -2270,14 +2302,23 @@ This section illustrates presentation of a Credential conformant to [@VC_DATA] t
 
 The Credential format identifiers are `ldp_vc` to request a W3C Verifiable Credential and `ldp_vp` to request a W3C Verifiable Presentation.
 
-Cipher suites should use signature suites names defined in [Linked Data Cryptographic Suite Registry](https://w3c-ccg.github.io/ld-cryptosuite-registry/).
```

Comment:
I removed this line as the registy is very outdated. I think it does more harm pointing to this than it does good.

Reply:
Found a new registry containing a list of securing mechanism specifications, so updated it to:

> Cipher suites should use Data Integrity compatible securing mechanisms defined in [Verifiable Credential Extensions](https://www.w3.org/TR/vc-extensions/#securing-mechanisms).